src/annotations/standard/PropertyAnnotation.php

```diff
@@ -14,12 +14,98 @@
 namespace mindplay\annotations\standard;
 
 use mindplay\annotations\Annotation;
+use mindplay\annotations\IAnnotationParser;
```

Comment:
Are you sure, that `use ` annotations are sorted alphabetically? The PhpStorm (if you use it) does that automatically usually.